packages/fxa-payments-server/src/components/AppLayout/index.tsx

```diff
@@ -47,6 +49,13 @@ export const SettingsLayout = ({
     return () => document.body.classList.remove('settings');
   }, [ children ]);
 
+  let breadcrumbs = (
+      <Breadcrumbs>
+        <li><a href="//127.0.0.1:3030/settings">Account Home</a></li>
```

Comment:
This is where you'll want to use the content server URL from config via AppContext